docker/scylla-sct/ubuntu/Dockerfile

```diff
@@ -12,18 +12,23 @@ SHELL ["/bin/bash", "-c"]
 #    00hzYw5m.HyAY
 #
 # For more details see man page for useradd(8)
+RUN apt-get update && apt-get install -y --no-install-recommends apt-transport-https gnupg2 software-properties-common curl
```

Comment:
we could save some space on this layer with `&&  rm -rf /var/lib/apt/lists/*`

Reply:
It's not that we upload this image anywhere...